services/web/server/src/simcore_service_webserver/remote_debug.py

```diff
@@ -25,15 +34,14 @@ def setup_remote_debugging(app: web.Application):
             #
             import ptvsd
 
-            REMOTE_DEBUGGING_PORT = 3000
             ptvsd.enable_attach(
```

Comment:
minore: now one uses debugpy instead of ptvsd (see [https://github.com/ITISFoundation/osparc-dask-gateway/blob/22d8663fc143ef3e76b4b397af1ec3ce4fcdd003/services/osparc-gateway-server/src/osparc_gateway_server/remote_debug.py](https://github.com/ITISFoundation/osparc-dask-gateway/blob/22d8663fc143ef3e76b4b397af1ec3ce4fcdd003/services/osparc-gateway-server/src/osparc_gateway_server/remote_debug.py))

Reply:
Good idea. I prefer doing that once for the entire repo in a separate PR. Added #2844